src/components/CharacterLimitMainText/components/CharacterLimit.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import { render } from '@testing-library/react';
+
+import CharacterLimit from './CharacterLimit';
+
+describe('CharacterLimit Component', () => {
+  it('should apply the "exceeded" class when character limit is exceeded', () => {
+    const maxLength = 10;
+    const value = '12345678901';
+    const svg = <svg />;
+
+    const { container } = render(
+      <CharacterLimit maxLength={maxLength} value={value} svg={svg} id={''} />
+    );
+
+    const characterLimitElement = container.querySelector('.characterLimit');
+    expect(characterLimitElement).toHaveClass('exceeded');
+
+    const remainingCharactersElement = container.querySelector(
+      '.characterLimit span'
+    );
+    if (remainingCharactersElement) {
+      expect(remainingCharactersElement.textContent).toBe('-1');
+    } else {
+    } //if null fuck u
+  });
+});
+
+it('should not apply the "exceeded" class when character limit is not exceeded', () => {
```

Comment:
Exatamente o mesmo comentário do bloco it estar sem um describe em volta.

Reply:
done